incubator/virtualcluster/cmd/kubectl-vc/util.go

```diff
@@ -94,6 +94,9 @@ func readFromFileOrURL(path string) ([]byte, error) {
 		defer resp.Body.Close()
 
 		yamlContent, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
This code base was moved to sigs.k8s.io/cluster-api-provider-nested/virtualcluster/cmd/kubectl-vc/util.go I would probably remove this file change as to not cause issues later when we finally delete that directory from this repo. 

/cc @Fei-Guo

Reply:
Yes, agree. You can submit a PR to cluster-api-provider-nested/virtualcluster repo instead.